### PR TITLE
Basic AWRVI dashboard view

### LINKED ISSUE #26: Build a basic AWRVI dashboard view

### LINKED ISSUE #71: Create the rails dev:prime task
This should programmatically generate index data for the system.

Have it randomly select 10% of the communities to generate indexes for.

And then have it generate a variety of indexes that are filled out, from fully complete to only partially complete

### PR DESCRIPTION
**Pull request by grimm**
Fixes #26 Fixes #71

### CODE REVIEW

**Comment by grimm** [comment]
@gina-alaska/awrvi 
Ready for review

File: app/models/category.rb [hunk 1]
```diff
@@ -8,4 +8,8 @@ class Category < ApplicationRecord
   def no_destroy
     false
   end
+
+  def to_s
```
Comment (teknofire) [inline app/models/category.rb:12]: Don't forget to add tests for methods you create in models

**Comment by teknofire** [comment]
I can't test this as there is no communities geojson file

**Comment by teknofire** [comment]
Getting `NoMethodError: undefined method`read_communities'`error when running rake`rake db:seeds`that method should probably be named`create_from_geojson`

**Comment by grimm** [comment]
@gina-alaska/awrvi 
Ready for review.

**Comment by grimm** [comment]
@gina-alaska/awrvi Ready for review

File: test/models/category_test.rb [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+  setup do
+    @category = categories(:one)
+  end
+
+  test "no_destroy should return false" do
+    refute @category.no_destroy
+  end
+
+  test "to_s must return category name" do
+    assert_equal @category.name, @category.to_s, "to_s failed and is not equal to name"
```
Comment (teknofire) [inline test/models/category_test.rb]: Shouldn't need the third option, the default message from `assert_equal` should be fine

Comment (grimm) [inline test/models/category_test.rb]: When I didn't put that in the error message was just, "test failed, no message" or something to that effect.

**Comment by grimm** [comment]
@gina-alaska/awrvi  Ready for review.

**Comment by teknofire** [comment]
:+1:

**Pull request merged**